operatorhub-resources/api-manager-operator/api-manager-operator.package.yaml

```diff
@@ -1,4 +1,5 @@
 packageName: api-manager-operator
```

Comment:
This has to change as wso2am-operator... Wherever you see api-manager-operator, change it as wso2am-operator